kubespawner/spawner.py

```diff
@@ -1724,6 +1724,7 @@ def _expand_user_properties(self, template):
             servername=safe_servername,
             unescaped_servername=raw_servername,
             hubnamespace=hub_namespace,
+            server_base_url=self.server.base_url,
```

Comment:
FWIW, I'd call this `base_url` since `server_base_url` for the server feels redundant to me.